settings.lua

```diff
@@ -2145,6 +2060,61 @@ function ReturnToFirstPointSetting:isReleaseDriverActive()
 	return self:get() == self.RELEASE_DRIVER or self:get() == self.RETURN_TO_START_AND_RELEASE_DRIVER
 end
 
+--- Generic offset setting with increment/decrement
+---@class OffsetSetting : FloatSetting
+OffsetSetting = CpObject(FloatSetting)
+function OffsetSetting:init(name, label, toolTip, vehicle, value)
+	FloatSetting.init(self, name, label, toolTip, vehicle, value)
+end
+
+-- increment/decrement offset
+function OffsetSetting:changeBy(changeBy)
+	self.value = courseplay:round(self.value, 1) + changeBy
+	if abs(self.value) < 0.1 then
+		self.value = 0
+	end
+end
+
```

Comment:
@pvaiko What is your opinion for using something like this "https://en.wikipedia.org/wiki/Floating-point_arithmetic" instead of using floats directly ? This would eliminate the need for rounding and we could use the base + exponent to represet the allowed change. `For example: 0,0.1,0.2,....,-0.1,-0.2,... we could use 0*0.1, 1*0.1, 2*0.1, -1*0.1, -2*0.1`
It would would also make mp synchronizing more save, as there are no rounding errror with integer.